Fixes #13426 - Change status colors to adapt to Patternfly palette

dLobatog opened the pull request:
The status colors are a bit off from the colors suggested in the palette.
https://www.patternfly.org/styles/color-palette/

ehelms commented:
You might consider switching to use of the variables, so that if the color is tweaked, these get tweaked to match (e.g. https://github.com/patternfly/patternfly-sass/blob/master/assets/stylesheets/patternfly/_variables.scss#L193)

shlomizadok commented:
+1 for color variables [test]

dLobatog commented:
Updated to use Patternfly variables

domcleal commented on app/assets/javascripts/components.js:
```diff
@@ -0,0 +1,2 @@
+//= require_tree ./components
```
Did you mean to add these files?

dLobatog replied:
On 02/16, Dominic Cleal wrote:

> > @@ -0,0 +1,2 @@
> > +//= require_tree ./components
> 
> Did you mean to add these files?
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/theforeman/foreman/pull/3129/files#r52993007
> Nope, my bad.

## 

Daniel Lobato Garcia

@dLobatog
blog.daniellobato.me
daniellobato.me

GPG: http://keys.gnupg.net/pks/lookup?op=get&search=0x7A92D6DD38D6DE30
Keybase: https://keybase.io/elobato

dLobatog committed:
Fixes #13426 - Change status colors to adapt to Patternfly palette

The status colors are a bit off from the colors suggested in the palette.
https://www.patternfly.org/styles/color-palette/

dLobatog commented:
OK, now it should just contain the css stylesheet change

domcleal commented:
Merged as 9db5b4f, thanks @dLobatog.
